We don't have pytest_relaxed and invoke packages is Solaris. Because of that,
raises decorator is implemented here.

Neither icecream nor invoke is available on Solaris, but thankfully, they
are not strictly required.

This is not intended for upstream.

--- paramiko-4.0.0/tests/test_client.py
+++ paramiko-4.0.0/tests/test_client.py
@@ -33,7 +33,15 @@ import weakref
 from tempfile import mkstemp
 
 import pytest
-from pytest_relaxed import raises
+def raises(expected):
+    def outer(func):
+        def inner(*args, **kwargs):
+            try:
+                func(*args, **kwargs)
+            except expected:
+                pass
+        return inner
+    return outer
 from unittest.mock import patch, Mock
 
 import paramiko
--- paramiko-4.0.0/tests/conftest.py
+++ paramiko-4.0.0/tests/conftest.py
@@ -4,7 +4,7 @@ import shutil
 import threading
 from pathlib import Path
 
-from invoke.vendor.lexicon import Lexicon
+#from invoke.vendor.lexicon import Lexicon
 
 import pytest
 from paramiko import (
@@ -20,12 +20,12 @@ from ._loop import LoopSocket
 from ._stub_sftp import StubServer, StubSFTPServer
 from ._util import _support
 
-from icecream import ic, install as install_ic
+#from icecream import ic, install as install_ic
 
 
 # Better print() for debugging - use ic()!
-install_ic()
-ic.configureOutput(includeContext=True)
+#install_ic()
+#ic.configureOutput(includeContext=True)
 
 
 # Perform logging by default; pytest will capture and thus hide it normally,
--- paramiko-4.0.0/pyproject.toml
+++ paramiko-4.0.0/pyproject.toml
@@ -11,7 +11,6 @@ authors = [{name = "Jeff Forcier", email
 dependencies = [
     "bcrypt>=3.2",
     "cryptography>=3.3",
-    "invoke>=2.0",
 ]
 optional-dependencies = {gssapi = [
     "pyasn1>=0.1.7",
